packages/typescript/src/logger.ts

```
@@ -313,6 +313,344 @@ try {
+    return _fs;
+  } catch {
+    return null;
+  }
+}
+
+/**
+ * Strip ANSI escape codes from a string for plain-text logging.
+ */
+function stripAnsi(str: string): string {
+  return str.replace(
+    /\x1B(?:\[[\x20-\x3F]*[\x40-\x7E]|\].*?(?:\x07|\x1B\\)|\(B)/g,
```

Biome (2.4.9)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)